### PR TITLE
Cypress/landing page testing

### PR DESCRIPTION
**Pull request by Ant-Shell**
# Pull Request

## Description:
Cypress testing for landing page which includes:
- Adding fixtures
- Testing Navbar
- Testing Featured Businesses
- Testing Footer
- Adding cypress env file
- Install eslinter package
- Install coverage package

### CODE REVIEW

**Review by coleanthony1990 — APPROVED** [review]
yeah!!!!

**Review by vfields — APPROVED** [review]
Reviewed with the group - thank you so much for this Anthony!

File: cypress/e2e/landing-spec.cy.js [hunk 1]
```diff
@@ -12,21 +12,12 @@ describe('Landing Page', () => {
     cy.visit('/')
   })
 
-/* Test banner contents 
-  - Test for logo
-  - Test for search bar
-  - Test for radio buttons
-  - Test for search button
-  - Test for welcome message (no user login)
-  - Test for "Home | Login | Favorites"
```
Comment (niksseif) [inline cypress/e2e/landing-spec.cy.js]: Love seeing this! Always remove commented out(dead code) before pushing up code.

File: cypress/e2e/landing-spec.cy.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 describe('Landing Page', () => {
   beforeEach(() => {
     cy.intercept('GET', Cypress.env('restaurant'), {
-      fixture: 'restaurantData.json'
+      fixture: 'singleRestaurantData.json'
```
Comment (niksseif) [inline cypress/e2e/landing-spec.cy.js:4]: Testing for singular dataset makes your test run faster, Nice work.

File: cypress/e2e/landing-spec.cy.js [hunk 3]
```diff
@@ -35,23 +35,24 @@ it('has an overview blurb', () => {
 })
 
 it('has featured local businesses', () => {
-  
+  cy.get('[data-cy="featured-cards"]')
+  .within(() => {
+    cy.get('[data-cy="business-card"]').first()
+    .should('contain', 'The Buff Restaurant').and('contain', 4.5)
+    .find('img').should('have.attr', 'src').should('include', 'https://s3-media2.fl.yelpcdn.com/bphoto/p95cYM_TgNOlHvw_cf0SAw/o.jpg')
```
Comment (niksseif) [inline cypress/e2e/landing-spec.cy.js:43]: Make sure to indicate on your README that your work is tested for accessibility as well.

**Review by niksseif — COMMENTED** [review]
Nice work! You have great atomic commits.

**Pull request merged**